src/components/common/with-authentication/withSamlSso.jsx

```diff
@@ -0,0 +1,9 @@
+import { withAuthentication } from './withAuthentication';
+
+const withSamlSso = WrappedComponent => (
+  props => <WrappedComponent
+    loginUrl={`${process.env.LMS_BASE_URL}/auth/idp_redirect/${process.env.IDP_SLUG}`}
+    {...props} />
+);
+
+export default WrappedComponent => withSamlSso(withAuthentication(WrappedComponent));
```

Comment:
While this is essentially how we have the `withSaml` component in its current form (a HOC wrapping another HOC), I'm wondering if it might make sense to start using the `compose` utility function from the library `recompose` to improve readability and maintainability of these authentication-related HOCs. See https://blog.logrocket.com/using-recompose-to-write-clean-higher-order-components-3019a6daf44c/ and https://github.com/acdlite/recompose/blob/master/docs/API.md#compose for more details.

That way, instead of writing `withSamlSso(withAuthentication(WrappedComponent))`, we could instead do:

```js
export default compose(
  withSamlSso,
  withAuthentication,
)(WrappedComponent);
```